File: src/my_project/easy_problems/from151To200/maximum_number_words.py
from typing import List 

class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:

        count: int = 0
        list_words: List[str] = text.split(' ')

        for word in list_words:
            for letter in brokenLetters:
                if letter in word:
                    count += 1
                    break
            

        return len(list_words) - count

solution = Solution()

print(solution.canBeTypedWords(text = "hello world", brokenLetters = "ad"))

print(solution.canBeTypedWords(text = "leet code", brokenLetters = "lt"))

print(solution.canBeTypedWords(text = "leet code", brokenLetters = "e"))

File: my_project_typescript/easy_problems_typescript/from51To100/maximumNumberWords.ts
function canBeTypedWords(text: string, brokenLetters: string): number {

    let count: number = 0
    let listWords: Array<string> =  text.split(" ")
    for (let word of listWords){
        for (let letter of brokenLetters){
            if (word.includes(letter)){
                count += 1
                break 
            }
        }
    }

    return listWords.length  - count 
};

console.log(canBeTypedWords("hello world", "ad"))

console.log(canBeTypedWords("leet code", "lt"))

console.log(canBeTypedWords("leet code", "e"))
